player/tests/data/quad.ron

```diff
@@ -0,0 +1,169 @@
+(
+    features: (bits: 0x0),
+    expectations: [
+        (
+            name: "Quad",
+            buffer: (index: 1, epoch: 1),
+            offset: 0,
+            data: File("quad.bin", 16384),
+        )
+    ],
+    actions: [
+        CreateShaderModule(
+            id: Id(0, 1, Empty),
+            data: "quad.vert.spv",
+        ),
+        CreateShaderModule(
+            id: Id(1, 1, Empty),
+            data: "quad.frag.spv",
+        ),
+        CreateTexture(Id(0, 1, Empty), (
+            label: Some("Output Texture"),
+            size: (
+                width: 64,
+                height: 64,
+                depth: 1,
+            ),
+            mip_level_count: 1,
+            sample_count: 1,
+            dimension: D2,
+            format: Rgba8Unorm,
+            usage: (
+                bits: 27,
+            ),
+        )),
+        CreateTextureView(
+            id: Id(0, 1, Empty),
+            parent_id: Id(0, 1, Empty),
+            desc: (
+                label: Some("Output Texture View"),
+                format: Some(Rgba8Unorm),
```

Comment:
Since these are the same as in the original, we can skip this and use `None` for both the format and `dimension`.